scripts/travis-publish.sh

```diff
@@ -20,10 +20,10 @@
 
 sbt_cmd="sbt ++$TRAVIS_SCALA_VERSION"
 
-export publish_cmd=""
+export publish_cmd="publishLocal"
```

Comment:
Is this right?

Reply:
I believe so. It tests the ability to publish artifacts locally. It was originally there but I mistakenly removed it as being redundant in my previous PR.